tests/test_release_transpiler_argument_types.py

```diff
@@ -0,0 +1,330 @@
+import os
+from os.path import abspath, dirname
+
+import pytest
+
+RELEASE_TRANSPILER_FILE_PATH = abspath(
+    dirname(dirname(abspath(__file__))) + "/komodo/release_transpiler.py"
+)
+VALID_RELEASE_FOLDER = abspath(dirname(__file__) + "/data/test_releases")
+VALID_RELEASE_BASE = "2020.01.a1"
+VALID_OVERRIDE_MAPPING_FILE = abspath(
+    dirname(dirname(__file__)) + "/examples/stable.yml"
+)  # "/Users/JONAK/Documents/FMU/SCOUT/komodo/examples/stable.yml"
+VALID_PYTHON_COORDS = "3.6,3.8"
+
+
+# TESTING COMBINER
+@pytest.mark.parametrize(
+    "args, expectedExitCode",
+    [
+        (
+            (
+                f"--release-folder {VALID_RELEASE_FOLDER} --release-base"
+                f" {VALID_RELEASE_BASE} --override-mapping"
+                f" {VALID_OVERRIDE_MAPPING_FILE} --py_coords 3.8,3.6"
+            ),
+            0,
+        ),
+        (
+            (
+                f"--release-folder {VALID_RELEASE_FOLDER} --release-base"
+                f" {VALID_RELEASE_BASE} --override-mapping"
+                f" {VALID_OVERRIDE_MAPPING_FILE}"
+            ),
+            0,
+        ),
+        (
+            (
+                f"--release-folder {VALID_RELEASE_FOLDER} --release-base"
+                f" {VALID_RELEASE_BASE} --override-mapping"
+                f" {VALID_OVERRIDE_MAPPING_FILE} --py_coords py3.8"
+            ),
+            512,
+        ),
+        (
+            (
+                f"--release-folder {VALID_RELEASE_FOLDER} --release-base"
+                f" {VALID_RELEASE_BASE} --override-mapping"
+                f" {VALID_OVERRIDE_MAPPING_FILE} --py_coords test"
+            ),
+            512,
+        ),
+        (
+            (
+                f"--release-folder {VALID_RELEASE_FOLDER} --release-base"
+                f" {VALID_RELEASE_BASE} --override-mapping"
+                f" {VALID_OVERRIDE_MAPPING_FILE} --py_coords 3.8, 3.6"
+            ),
+            512,
+        ),
+        (
+            (
+                f"--release-folder {VALID_RELEASE_FOLDER} --release-base"
+                f" {VALID_RELEASE_BASE} --override-mapping"
+                f" {VALID_OVERRIDE_MAPPING_FILE} --py_coords true"
+            ),
+            512,
+        ),
+    ],
+)
+def test_combine_py_coords_type(args, expectedExitCode):
+    exit_code = os.system(f"{RELEASE_TRANSPILER_FILE_PATH} combine {args}")
```

Comment:
When running the test code, you can assume that komodo has been pip-installed. That means that the transpiler has already been installed in PATH (as defined in pyproject.toml, with the name `komodo-transpiler`) (then you don't need to fiddle around with the file path to the python file).

Another way this is often accomplished is as in `test_cli.py` where `sys.argv` is modified, and then the `main()` function in the transpiler file is called as a Python function. Then we don't' have the overhead of running the the transpiler in a separate process (and the code in `test_cli.py` would be even prettier if it used a mock for sys.argv, and not modifying it directly).